setup.py

```diff
@@ -0,0 +1,21 @@
+# Licensed under a 3-clause BSD style license - see License.txt
+
+from setuptools import setup
+
+setup(
+    name='firefly_client',
+    version='1.0.0.b2',
+    description='Python API for Firefly',
+    author='IPAC LSST SUIT',
+    license='BSD',
+    url='http://github.com/Caltech-IPAC/firefly',
```

Comment:
Good catch. Looking at the pip packaging instructions, this should point to the firefly_client repository.